### PR TITLE
Add consolidated payment indicator

### PR DESCRIPTION
**Pull request by luoshan654**
Add consolidated payment indicator

### CODE REVIEW

**Comment by dindinw** [comment]
Please also fix the magic numbers introduced by https://github.com/Qitmeer/qitmeer-wallet/commit/83e5ebd6143f4523bfecdb951c6ccbf1bc4a89cb#diff-c35d9ca3f3d9de083c045b115be2f0ee 
The code introduced by this commit is hard to maintain. Please avoid code smells
@5dao

File: wallet/wallet.go [hunk 1]
```diff
@@ -1520,6 +1521,7 @@ b:
 	if err != nil {
 		return nil, err
 	} else {
+		msg=strings.ReplaceAll(msg,"\"","")
```
Comment (dindinw) [inline wallet/wallet.go:1524]: Why we have to replace the msg?

Comment (luoshan654) [reply wallet/wallet.go:1524]: The return after sending the transaction is like this "abfc5f36a4e17081fc5167f390e9f0575293892c7cc8e3ba098f55f1a3735b1a" 
Quotes need to be removed

Comment (dindinw) [reply wallet/wallet.go:1524]: Please refactor code after PR15, it's kind of unnecessary data handling because of poorly-designed data exchanging

**Review by dindinw — APPROVED** [review]

**Pull request merged**